.travis.yml

```diff
@@ -1,23 +1,27 @@
 language: python
 sudo: false
+
 python:
- - "2.7"
- - "3.4"
- - "3.5"
- - "3.6"
+ - 3.4
+ - 3.5
+ - 3.6
+
 env:
- - DJANGO_VERSION=1.9.11
- - DJANGO_VERSION=1.10.5
- - DJANGO_VERSION=1.11.2
+ - DJANGO_VERSION=1.11.14
+ - DJANGO_VERSION=2.0.7
```

Comment:
so this PR is both dropping Python 2 support and adding Django 2 support?

Reply:
this PR builds on top of the PR for django2, you could merge this one and two PRs for one click :smile: